kops/setup-efs.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+"""
+Hacky script to automate EFS setup for a given kops cluster
+
+1. Create an EFS file system
+2. Create a mount target with correct security groups & subnets for
+   a given kops cluster
+"""
+
+import sys
+import boto3
+import secrets
+import time
+
+def find_subnets(cluster_name, region):
+    """
+    Find all the subnets created by kops for this cluster
+    """
+    ec2 = boto3.client('ec2', region_name=region)
+    return ec2.describe_subnets(Filters=[
+        {'Name':'tag:KubernetesCluster', 'Values': [cluster_name]}
+    ])['Subnets']
+
+def find_security_groups(cluster_name, region):
+    """
+    Find security groups of master and nodes
+    """
+    ec2 = boto3.client('ec2', region_name=region)
+    return ec2.describe_security_groups(Filters=[
+        {'Name':'tag:Name', 'Values': [f'{t}.{cluster_name}' for t in ['masters', 'nodes']]}
```

Comment:
Do you really need the SG associated with `masters`?

Reply:
I checked - the 'masters' is for all master nodes. Since the share creator runs there, we should probably keep that.

The alternative is to do one EFS per hub, but I don't know if I entirely like that.

Reply:
I've added some comments to the script.

Reply:
> The alternative is to do one EFS per hub, but I don't know if I entirely like that.

Having an EFS per hub could be something needed from a security point of view, maybe?
Btw, I do think it is a conversation for later (not something we need to figure it out for this PR).